azure-pipelines/end-to-end-tests-dir/post-build-checks.ps1

```diff
@@ -966,11 +964,9 @@ foreach ($bad_dir in @('build-dir', 'downloads', 'installed-root', 'package-dir'
 
 $expectedFooter = @"
 $($PortfilePath): note: Adding a call to ``vcpkg_fixup_pkgconfig()`` may fix absolute paths in .pc files
-note: Absolute paths were found in the following files
-$($packagesRoot)$($NativeSlash)vcpkg-policy-absolute-paths_$($Triplet): note: the files are relative to `${CURRENT_PACKAGES_DIR} here
-note: include/vcpkg-policy-absolute-paths.h
```

Comment:
I am a bit sad to lose the form that a user would say in `portfile.cmake` in order to edit one of these files but making the paths clickable is probably worth it anyway...

Reply:
I can see arguments either way... though doesn't this apply to most post-lint checks? Maybe it should be centralized (printed once at the beginning or end of the post-lint checks if any fired)